public-ui/src/app/data-browser/views/ehr-view/ehr-view-react.component.tsx

```diff
@@ -239,14 +240,9 @@ h5.secondary-display {
     margin-left: 0.3em;
 }
 `;
-
-interface Props {
-    domainId: string;
-    searchTerm: string;
-}
-
 interface State {
     domain: any;
+    domainId: string;
```

Comment:
We should get this from the `id` property of `urlParamsStore` instead of adding another state variable.